src/backend/common/helpers/insights_leaderboard_match_helper.py

```diff
@@ -91,25 +91,26 @@ def _2025_most_coral_scored(
         if arguments.year != 2025:
             return None
 
+        def get_coral_count(sb: Dict) -> int:
+            return sum(
```

Comment:
is `sum([])` better than `+`?

Reply:
makes no difference, just less repetitive imo